aws-crt/IO/ClientBootstrap.cs

```diff
@@ -31,14 +31,13 @@ protected override bool ReleaseHandle()
 
         internal Handle NativeHandle { get; private set; }
 
-        private EventLoopGroup eventLoopGroup;
-        private HostResolver hostResolver;
-
         public ClientBootstrap(EventLoopGroup eventLoopGroup, HostResolver hostResolver = null)
         {
-            this.eventLoopGroup = eventLoopGroup;
-            this.hostResolver = hostResolver != null ? hostResolver : new DefaultHostResolver(eventLoopGroup);
-            NativeHandle = API.make_new(eventLoopGroup.NativeHandle.DangerousGetHandle(), this.hostResolver.NativeHandle.DangerousGetHandle());
+            if (hostResolver == null) {
```

Comment:
```csharp
hostResolver = new DefaultHostResolver(eventLoopGroup)
```
 in the signature, does that work?

Reply:
Doesn't appear to, eventLoopGroup can't be referred to